src/librustc_mir/borrow_check/nll/constraints/graph.rs

```diff
@@ -51,12 +51,12 @@ impl ConstraintGraph {
     }
 }
 
-crate struct Edges<'s> {
-    graph: &'s ConstraintGraph,
+crate struct Edges<'graph> {
+    graph: &'graph ConstraintGraph,
     pointer: Option<ConstraintIndex>,
 }
 
-impl<'s> Iterator for Edges<'s> {
+impl Iterator for Edges<'graph> {
```

Comment:
Do you need to use a named lifetime here? I don't see the lifetime used in the `impl`, so it seems like you should be able to use `'_`.

Reply:
I'm not sure. Part of me wants to use `'_` but I somewhat think this is an unanswered question; see https://mark-simulacrum.github.io/2018/07/19/inband-lifetimes.html#unused-lifetimes-in-impl-blocks as well.